packages/devtools_app/lib/src/http/http_request_data.dart

```diff
@@ -135,32 +132,38 @@ class HttpRequestData {
 
   /// A map of general information associated with an HTTP request.
   Map<String, dynamic> get general {
```

Comment:
cache the value here so we stop creating a ton of copies.  Otherwise the code in getters like `port` is a bit wasteful as we generate the map twice and the only use a bool from it.

Reply:
done